chart/values.yaml

```diff
@@ -2,6 +2,10 @@
 # which run inside the prometheus-benchmark - e.g. vmagent, vmalert, vmsingle.
 vmtag: "v1.102.1"
 
+# Controls whether to deploy a built-in vmsingle for testing
+# Useful for testing remote systems when vmsingle is not needed
+deployTSDB: true
```

Comment:
The option sounds weird. AFAIK, vmsingle depliyment here exists only for internal monitoring. It is not related to testing as is, only for collecting metrics from vmalerts and vmagents. So it is more like `disableMonitoring` option.

Reply:
Thanks for noticing, updated.